R/convergence.R

```diff
@@ -570,3 +708,26 @@ fold_draws <- function(x) {
 should_return_NA <- function(x) {
   anyNA(x) || checkmate::anyInfinite(x) || is_constant(x)
 }
+
+# splits chains into halves within a `draws_df` object
```

Comment:
Thank you for writing this method! I will make `split_chains` a separate generic but this can also be done after this PR has been merged.

Reply:
Great.